libs/remix-tests/src/runTestSources.ts

```diff
@@ -5,133 +5,142 @@ import { deployAll } from './deployer'
 import { runTest } from './testRunner'
 
 import Web3 from 'web3'
+import EventManager from './lib/eventManager'
```

Comment:
Would it be possible to use `const EventEmitter = require('events')` instead?

Reply:
will check

Reply:
done